_posts/en/newsletters/2024-12-13-newsletter.md

```diff
@@ -223,7 +223,19 @@ answers posted since our last update.*
 {% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME:bitschmidty
+- [How long does Bitcoin Core store forked chains?]({{bse}}124973)
+  Pieter Wuille explains that with the exception of Bitcoin Core nodes running
+  in pruned mode, blocks that a node downloads, regardless of if they are in the
+  main chain or not, are stored indefinitely.
+
+- [What is the point of solo mining pools?]({{bse}}124926)
+  Murch outlines why a Bitcoin miner might use a mining pool that doesn't divide
+  mining rewards to its particpants, a "solo mining pool".
```

Comment:
```suggestion
  mining rewards to its participants, a _solo mining pool_.
```
Also maybe consider italicizing instead of quoting, since this is a definition.